graphql/resolvers.go

```diff
@@ -4084,6 +4084,20 @@ func (r *ticketFieldsResolver) ResolutionName(ctx context.Context, obj *thirdpar
 
 func (r *Resolver) TicketFields() TicketFieldsResolver { return &ticketFieldsResolver{r} }
 
+func (r *taskResolver) Ami(ctx context.Context, obj *restModel.APITask) (*string, error) {
+	if obj.HostId != nil {
+		host, err := host.FindOneId(utility.FromStringPtr(obj.HostId))
+		if err != nil {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding host %s: '%s'", *obj.HostId, err.Error()))
```

Comment:
FromStringPtr (could just set the hostId variable at the top of the function and use throughout)